cclib/parser/serenityparser.py

```diff
@@ -52,6 +53,17 @@ def after_parsing(self):
 
         super().after_parsing()
 
+    def convert_to_spin(self, symbol):
```

Comment:
If you don't need the flexibility of a function, another way of doing it is https://github.com/cclib/cclib/blob/ddcf48ea79019427e93103cdcb39930ddbede66d/cclib/parser/orcaparser.py#L1306-L1331.  But I don't think it would be possible to as cleanly handle the error case, so the function is nicer in that regard.